ga4gh/datamodel/variants.py

```diff
@@ -415,6 +427,28 @@ def convertVariant(self, record, callSetIds):
             sampleIterator += 1  # REMOVAL
         return variant
 
+    def getVariant(self, compoundId):
+        if compoundId.chrom in self._chromFileMap:
+            varFileName = self._chromFileMap[compoundId.chrom]
+        else:
+            raise exceptions.ObjectNotFoundException(compoundId)
+        pos = int(compoundId.pos)
+        referenceName, startPosition, endPosition = \
+            self.sanitizeVariantFileFetch(
+                compoundId.chrom, pos, None)
+        # TODO vcf.fetch's startPosition arg seems to be exclusive,
+        # not inclusive!!!
+        startPosition = startPosition - 0.1
+        cursor = self.getFileHandle(varFileName).fetch(
+            referenceName, startPosition, endPosition)
+        for record in cursor:
+            if pos != record.pos:
+                continue
+            digest = hashlib.md5(record.ref + str(record.alts)).hexdigest()
```

Comment:
We should calculate the digest in one place --- a method on the relevant class would seem like the best option.